README.md

```diff
@@ -137,22 +137,38 @@ whole *Integration Service* product suite, there are some specific flags which a
   ~/is_ws$ colcon build --cmake-args -DBUILD_ROS1_TESTS=ON
   ```
 
-* `MIX_ROS_PACKAGES`: It accepts as an argument a list of [ROS 1 packages](https://index.ros.org/packages/),
+* `MIX_ROS_PACKAGES`: It accepts as an argument a list of [ROS packages](https://index.ros.org/packages/),
   such as `std_msgs`, `geometry_msgs`, `sensor_msgs`, `nav_msgs`... for which the required transformation
   library to convert the specific ROS 1 type definitions into *xTypes*, and the other way around, will be built.
-  This library is also known within the *Integration Service* context as `Middleware Interface Extension`
-  or `mix` library.
+  This list is shared with the [ROS 2 System Handle](https://github.com/eProsima/ROS2-SH#compilation-flags),
+  meaning that the ROS packages specified in the `MIX_ROS_PACKAGES` variable will also be built for *ROS 2*
+  if this *System Handle* is present within the *Integration Service* workspace. If a certain package
+  is only present in *ROS 1*, the `MIX_ROS1_PACKAGES` flag must be used instead.
```

Comment:
```suggestion
  if the corresponding *System Handle* is present within the *Integration Service* workspace. To avoid errors, if a certain package
  is only present in *ROS 1*, the `MIX_ROS1_PACKAGES` flag must be used instead.
```